src/App.js

```diff
@@ -20,8 +19,7 @@ const App = () => (
         <Route path="/" element={<Splash />} />
         <Route path="/login" element={<Login />} />
         <Route path="/signup" element={<SignUp />} />
-        <Route path="/Rooms/:id" element={<Detail />} />
-        <Route path="/Rooms/*" element={<Rooms />} />
+        <Route path="/Rooms/" element={<Rooms />} />
```

Comment:
Kindly change the path to "/Rooms/*"

Reply:
Thank @AxelSoler, for the review. the thing is if I leave the `/*` in the rooms route when we display the detail for the single room the rooms will remain in place. but we can discuss this further on zoom.